data/2022_23/seminare/P-MAT/kockaty_vykend.yml

```diff
@@ -0,0 +1,34 @@
+# Nazov udalosti
+name: Kockatý víkend
+
+# Vedne obory (1+) pre danu akciu. Mozne hodnoty: mat, fyz, inf a other.
+sciences:
+  - mat,fyz
```

Comment:
Ako spravím, keď sa to týka 2 vedných disciplín?

Reply:
je to zoznam takže
```suggestion
  - mat
  - fyz
```